src/generic/stage2_build_tape.h

```diff
@@ -151,7 +128,7 @@ struct structural_parser {
   ) : structurals(buf, len, _doc_parser.structural_indexes.get(), next_structural), doc_parser{_doc_parser}, depth{0} {}
 
   WARN_UNUSED really_inline bool start_scope(internal::tape_type type, ret_address continue_state) {
-    doc_parser.containing_scope[depth].tape_index = doc_parser.current_loc;
+    doc_parser.containing_scope[depth].tape_index = uint32_t(doc_parser.next_loc - doc_parser.doc.tape.get());
```

Comment:
You have made this change about how we write to the tape along with changing the number parsing API... I do not object to it, but it makes it a bit harder to see what hits us and where.

Reply:
It's actually in [one commit](https://github.com/simdjson/simdjson/pull/884/commits/2821dc12d04850bf33f2952210a30569477540c7) separate from the number parsing API change, so that impact can be measured. When building up progress towards a meaningful, I try to make sure I can walk back through the commits and re-benchmark so I know exactly what had what effect. I also try to fix the bugs in the commit they were introduced, usually, so that the benchmarks stay valid.

Also so I can peel them off and use them in other attempts :)